resource_management_mobile/src/app/auth/login/login.page.scss

```diff
@@ -0,0 +1,83 @@
+
+$font-weight:600;
+$curve-size:47px;
+@mixin flexDisplay {
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    align-items: center;
+}
+
+
```

Comment:
Format the code - keep it better visual and clean

Reply:
Formatted